src/libs/runtime-utils.js

```diff
@@ -10,8 +10,8 @@ import * as Style from 'src/libs/style'
 import * as Utils from 'src/libs/utils'
 
 
-export const defaultDataprocDiskSize = 60 // For both main and worker machine disks. Dataproc clusters don't have persistent disks.
-export const defaultGceBootDiskSize = 70 // GCE boot disk size is not customizable by users. We use this for cost estimate calculations only.
+export const defaultDataprocDiskSize = 80 // For both main and worker machine disks. Dataproc clusters don't have persistent disks.
+export const defaultGceBootDiskSize = 80 // GCE boot disk size is not customizable by users. We use this for cost estimate calculations only.
```

Comment:
I think UI does actually get these defaults from leo...these values are only used if leo doesn't return the value properly

Reply:
Oh hm these aren't passed in the Leo `createRuntime` request?

In any case this should be okay since I also updated to 80g in Leo `reference.conf`..

Reply:
`defaultGceBootDiskSize` isn't passed in since it's internally set by Leo. It is used for cost estimate calculations so it's good that we're updating it to reflect the new value Leo will start using.

`defaultDataprocDiskSize` determines the default disk size displayed on the runtime modal when there is no existing runtime. It also sets the minimum. So, also good that we're updating it here.

It is true that when there _is_ an existing runtime, the UI uses the values coming from Leo.

Reply:
yea...definitely still good to update..